xml/deployment_prep_aarch64_boot.xml

```diff
@@ -17,145 +17,7 @@
-</screen>
-  <para>
-   This means that screen listens to the first serial port with a baud rate of
-   115200. From this point on, the installation proceeds similarly to the
-   text-based installation over this terminal.
-  </para>
- </sect2>
-
- <sect2 xml:id="sec.prep.boot.aarch64.ssh">
-  <title>Installation with SSH</title>
-  <para>
-   If you do not have direct access to the machine and the installation must be
-   initiated from a management console, you can control the
-   entire installation process over the network. To do this, enter the
-   parameters <option>ssh=1</option> and
-   <option>ssh.password=<replaceable>SECRET</replaceable></option> at the boot
-   prompt. An SSH daemon is then launched in the system and you can log in as
-   user &rootuser; with the password <replaceable>SECRET</replaceable>.
-  </para>
-  <para>
-   To connect, use <command>ssh -X</command>. X-Forwarding over SSH is
-   supported, if you have a local X server available. Otherwise, &yast;
-   provides a text interface over ncurses. &yast; then guides you through the
-   installation. This procedure is described in detail in
-   <xref
-        linkend="sec.deployment.remoteinst.scenario.sshdyn"/>.
-  </para>
-  <para>
-   If you do not have a DHCP server available in your local network, manually
-   assign an IP address to the installation system. Do this by entering the
-   option <option>HostIP=<replaceable>IPADDR</replaceable></option> at the boot
-   prompt.
-  </para>
- </sect2>
-
- <sect2 xml:id="sec.prep.boot.aarch64.vnc">
-  <title>Installation over VNC</title>
-  <para>
-   If you do not have direct access to the system, but want a graphical
-   installation, install &productname; over VNC. This method is described in
-   detail in <xref linkend="sec.deployment.remoteinst.monitor.vnc"/>.
-  </para>
-  <para>
-   As suitable VNC clients are also available for other operating systems, such
-   as Microsoft Windows and mac OS, the installation can also be controlled from
-   computers running those operating systems.
-  </para>
- </sect2>
-
- <sect2 xml:id="sec.prep.boot.aarch64.autoyast">
-  <title>Installation with &ay;</title>
-  <para>
-   If you need to install &productname; on several computers with similar
-   hardware, it is recommended you perform the installations with the aid of
-   &ay;. In this case, start by installing one &productname; and use this to
-   create the necessary &ay; configuration files.
-  </para>
-  <para os="sles,osuse">
-   &ay; is extensively documented in <xref linkend="book.autoyast"/> .
-  </para>
-  <para os="sled">
-   &ay; is extensively documented at
-   <link xlink:href="https://www.suse.com/documentation/sles-12/book_autoyast/data/book_autoyast.html"/>
-   .
-  </para>
- </sect2>
+ <xi:include xpointer="element(/1/3)" href="deployment_prep_x86_boot.xml"/>
+ <xi:include xpointer="element(/1/4)" href="deployment_prep_x86_boot.xml"/>
+ <xi:include xpointer="element(/1/5)" href="deployment_prep_x86_boot.xml"/>
```

Comment:
Are you sure this works? What happens if someone changes deployment_prep_x86_boot.xml, e.g. by adding block elements?

Reply:
Yes this works, but yes, if someone adds a new para, then this may produce unwanted results. I'll check if we can get an xpointer that includes all para. But just in case that's why there is a XML comment with a warning at the beginning of this file.